draft-ietf-quic-recovery.md

```diff
@@ -1574,6 +1574,29 @@ DetectAndRemoveLostPackets(pn_space):
 ~~~
 
 
+## Upon Dropping Initial or Handshake Keys
+
+When Initial or Handshake keys are discarded, packets from the space
+are discarded and loss detection state is updated.
+
+Pseudocode for OnPacketNumberSpaceDiscarded follows:
+
+~~~
+OnPacketNumberSpaceDiscarded(pn_space):
+  assert(pn_space != ApplicationData)
+  // Remove any unacknowledged packets from flight.
+  foreach packet in sent_packets[pn_space]:
+    if packet.in_flight
+      bytes_in_flight -= size
```

Comment:
This was placed below because it modifies bytes_in_flight, but it also modifies a bunch of loss detection state, so I don't have a strong opinion on where it is.

Reply:
Hmm, I missed that.  This argues for dual sections.  I'll work on it.